Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.12-slim
+LABEL maintainer="testtest@gmail.com"
+
+ENV PYTHONUNBUFFERED=1
+
+COPY requirements.txt requirements.txt
```

Comment:
Consider using a multi-stage build or a virtual environment to install dependencies to reduce the size of the final image and avoid potential conflicts with system packages.